frontend/src/components/Admin/TeamEvent/TeamEventDetails.tsx

```diff
@@ -137,18 +139,62 @@ const TeamEventDetails: React.FC = () => {
         )}
       </div>
 
-      <div className={styles.listsContainer}>
-        <div className={styles.listContainer}>
-          <h2 className={styles.memberTitle}>Members Pending</h2>
-          <AttendanceDisplay status={'pending' as Status} teamEvent={teamEvent} />
+      <div className={styles.row_direction}>
+        <div className={styles.dropdown}>
+          <label className={styles.bold}>Select Status:</label>
+          <Dropdown
+            placeholder="Select Status"
+            fluid
+            selection
+            options={allStatus.map((status) => ({
+              text: status,
+              value: status
+            }))}
+            onChange={(_, data) => {
+              setStatus(data.value as Status);
+            }}
+          />
         </div>
 
-        <div className={styles.listContainer}>
-          <h2 className={styles.memberTitle}>Members Approved</h2>
-          <AttendanceDisplay status={'approved' as Status} teamEvent={teamEvent} />
-          <h2 className={styles.memberTitle}>Members Rejected</h2>
-          <AttendanceDisplay status={'rejected' as Status} teamEvent={teamEvent} />
-        </div>
+        {status && (
+          <div className={styles.inline}>
+            <Button
+              onClick={() => {
+                setStatus(undefined);
+                fullReset();
```

Comment:
Hm, why are we calling a `fullReset` here? I know it's not fetching data, but that's what it feels like to the user when you hit the button. Can we write this so that we don't need to see the loading wheel when we hit the button?

Reply:
I agree that loading the entire page is unnecessary. Initially, `fullReset` was called to reset the dropdown to "Select Status" after clicking the "View All" button, but I found a way to do that without loading the entire page again.